src/hb-ot-color-sbix-table.hh

```diff
@@ -27,6 +27,12 @@
 
 #include "hb-open-type-private.hh"
 
+/*
+ * sbix -- Standard Bitmap Graphics Table
+ * https://docs.microsoft.com/en-us/typography/opentype/spec/sbix
+ * https://developer.apple.com/fonts/TrueType-Reference-Manual/RM06/Chap6sbix.html
```

Comment:
There are more things on Apple's specification but currently no fonts on macOS to test them.

Reply:
We'll just support the OpenType version. The rest is deprecated and there are no fonts out there.